packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -344,6 +344,12 @@ class Navigator extends StatefulWidget {
   }
 
   /// The state from the closest instance of this class that encloses the given context.
+  ///
+  /// Typical usage is as follows:
+  ///
+  /// ```dart
+  /// NavigatorState navigator = Navigator.of(context);
```

Comment:
You might want to have the same Builder-related text here for folks who are above MaterialApp